### ISSUE TITLE
Blueprint version mismatch

### ISSUE THREAD

**Issue by JohanEikelenboom** [Blackymas/NSPanel_HA_Blueprint#1077]
Just tried to update my NSPanel to the latest version. At the moment it is stuck at the bootscreen where it shows:
TFT: 4.0
ESPHome: 4.0
Blueprint: Retry: <counter>

In the ESP logging I found the following error:
[script.check_versions:2229] | Blueprint version mismatch!

If I look in the blueprint, it says it's version 4.0

I'm running HAOS which is on the latest version (HAOS version 10.5, HA version 2023.9.2)

**Comment by plasterer41** [comment]
Did you reboot home assistant?Sent from my iPhoneOn 22 Sep 2023, at 19:23, JohanEikelenboom ***@***.***> wrote:﻿
Just tried to update my NSPanel to the latest version. At the moment it is stuck at the bootscreen where it shows:
TFT: 4.0
ESPHome: 4.0
Blueprint: Retry: 
In the ESP logging I found the following error:
[script.check_versions:2229] | Blueprint version mismatch!
If I look in the blueprint, it says it's version 4.0
I'm running HAOS which is on the latest version (HAOS version 10.5, HA version 2023.9.2)

**Comment by JohanEikelenboom** [comment]
Rebooted HA several times, re-installed ESPHome several times, re-downloaded the blueprint several times

**Comment by edwardtfn** [comment]
Take a look at Home Assistant logs (Settings > System > Logs). Can you see any error message related to this automation/blueprint?

**Comment by edwardtfn** [comment]
Also, on the Blueprint settings, make sure the panel is selected properly. It is the very first input in the Blueprint.

**Comment by JohanEikelenboom** [comment]
Looks like it is solved. Don't know if it was a user error or something in the blueprint but I noticed the following (after looking a couple of times). This is what happend:
I named the NSPanel **nspanel_1** but this showed up as **nspanel 1** (without the underscore) in the blueprint. I recreated the ESPHome config and named it **nspanel** and uploaded it through the webinterface of the NSPanel.

After a restart of HA, the blueprint picked up the correct name and now it looks like it's working as expected (at least I get the home screen)
